test/v200/utils/common/component_test_utils.go

```diff
@@ -0,0 +1,172 @@
+package common
+
+import (
+	"fmt"
+	"strconv"
+
+	schema "github.com/devfile/api/v2/pkg/apis/workspaces/v1alpha2"
+)
+
+// componentAdded adds a new component to the test schema data and to the parser data
+func (devfile *TestDevfile) componentAdded(component schema.Component) {
+	LogInfoMessage(fmt.Sprintf("component added Name: %s", component.Name))
+	devfile.SchemaDevFile.Components = append(devfile.SchemaDevFile.Components, component)
+	if devfile.Follower != nil {
+		devfile.Follower.AddComponent(component)
+	}
+}
+
+// componetUpdated updates a component in the parser data
+func (devfile *TestDevfile) componentUpdated(component schema.Component) {
+	LogInfoMessage(fmt.Sprintf("component updated Name: %s", component.Name))
+	if devfile.Follower != nil {
+		devfile.Follower.UpdateComponent(component)
+	}
+}
+
+// addVolume returns volumeMounts in a schema structure based on a specified number of volumes
+func (devfile *TestDevfile) addVolume(numVols int) []schema.VolumeMount {
```

Comment:
would rename `(devfile *TestDevfile)` to `(testDevFile *TestDevfile)` to be consistent with the commands file

Reply:
Have notice the inconstancy, will resolve this as the test develops.